client/src/components/pages/index.ts

```diff
@@ -9,6 +9,7 @@ export { default as SignInPage } from './sign-in';
 export { ClansPage, ClanPage } from './clans';
 export { default as SignUpPage } from './sign-up';
 export { default as HomePage } from './home';
+export { LandingPage as LandingPageComponent } from './landing';
```

Comment:
How does it make sense to use an extended name here?

Reply:
I have a container called LandingPage so I needed a new name. Maybe it's better to rename container, idk

Reply:
I agree with @AlexanderKhapchenko, it would be better to name all components in the same style